ImageSharpCompare/ImageSharpCompare.cs

```diff
@@ -82,20 +82,45 @@ public static bool ImagesAreEqual(Stream actual, Stream expected)
         /// <param name="expected"></param>
         /// <returns>True if every pixel of actual is equal to expected</returns>
         public static bool ImagesAreEqual(Image actual, Image expected)
+        {
```

Comment:
Wouldnt overloads also solve the issue, where every Image<Rgb24> wont be disposed while Image could still be diposed using "using"s?

Reply:
That would be more readable and wont trigger any CA2000 compiler warning.

Reply:
CA should be happy now